src/options/settings/components/container.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+
+import SearchInjection from './SearchInjection'
+import { getLocalStorage, setLocalStorage } from 'src/search-injection/utils'
+import { SEARCH_INJECTION_KEY } from 'src/search-injection/constants'
+
+class Container extends React.Component {
```

Comment:
Calling the class just `Container` means you have to open it up and look inside to see what it does, as the name doesn't give much info. You also might want to add more containers in this dir later for whatever reason. Maybe `SearchInjectionContainer` would be better as it simply wraps around that view.